Frontend/ui-customer/src/Screens/ZooTicketBookingFlow/TicketBookingScreen/Controller.purs

```diff
@@ -51,44 +59,66 @@ data Action = AfterRender
             | IncrementSliderIndex
             | DecrementSliderIndex
             | PaymentStatusAction String
-            | Copy
             | GoHome
             | RefreshStatusAC PrimaryButton.Action
+            | SelectDestination TicketCategoriesOptionData
+            | SelectSlot String SlotInterval
+            | OpenGoogleMap (Maybe Number) (Maybe Number)
+            | Copy String
 
 data ScreenOutput = GoToHomeScreen TicketBookingScreenState
                   | GoToTicketPayment TicketBookingScreenState
                   | GoToGetBookingInfo TicketBookingScreenState BookingStatus
                   | TryAgain TicketBookingScreenState
                   | RefreshPaymentStatus TicketBookingScreenState
+                  | BookTickets TicketBookingScreenState
+                  | GoToOpenGoogleMaps TicketBookingScreenState Number Number
 
 eval :: Action -> TicketBookingScreenState -> Eval Action ScreenOutput TicketBookingScreenState
+
+eval AfterRender state = continue state
```

Comment:
This is not required, This line can be removed

Reply:
done